vm/src/utils.rs

```diff
@@ -263,12 +273,14 @@ pub mod test_utils {
             CairoRunner::new(&program, $layout.to_string(), proof_mode).unwrap()
         };
     }
-    pub(crate) use cairo_runner;
+    pub use cairo_runner;
 
-    pub(crate) use crate::stdlib::{collections::BTreeMap, sync::Arc};
+    pub use crate::stdlib::{collections::BTreeMap, sync::Arc};
     pub(crate) use crate::types::program::HintsCollection;
-    pub(crate) use crate::types::program::Program;
+    pub use crate::types::program::Program;
```

Comment:
`Program` is already public, we should use the full path in the macro rather than re-exporting it here